src/object/schema_manager.c

```diff
@@ -2828,7 +2828,8 @@ sm_mark_system_class_for_catalog (void)
     CTV_METHARG_NAME, CTV_METHARG_SD_NAME, CTV_METHFILE_NAME,
     CTV_INDEX_NAME, CTV_INDEXKEY_NAME, CTV_AUTH_NAME,
     CTV_TRIGGER_NAME, CTV_STORED_PROC_NAME, CTV_STORED_PROC_ARGS_NAME,
-    CTV_PARTITION_NAME, CT_COLLATION_NAME, NULL
+    CTV_PARTITION_NAME, CT_COLLATION_NAME, CT_DB_SERVER_NAME,
+    CTV_DB_SERVER_NAME, NULL
```

Comment:
How about changing it to list only one name per line with this change?

Reply:
OK, I agree.